src/Features/header/components/MobileApplicationPromo/MobileApplicationPromo.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+
+import { MobileApplicationPromo } from './MobileApplicationPromo';
+
+describe('Test MobileApplicationPromo component', () => {
```

Comment:
the description of `describe` block contains only component name, MobileApplicationPromo - in this case